website/docs/static/luigi-client/luigi-element.js

```diff
@@ -0,0 +1,55 @@
+/**

```

Comment:
I don't see how or why the **website/docs/static/luigi-client/*** folder contents should be included here. The files in there should be inherited from **node_modules** as they are dynamically changing each time we update luigi version in website package.json. I would not expect any such files to be committed. Can you please elaborate why its here ?